src/core/algorithms/md/hymd/preprocessing/similarity_measure/smith_waterman_gotoh.h

```diff
@@ -3,6 +3,6 @@
 #include <string>
 
 namespace algos::hymd::preprocessing::similarity_measure {
-double NormalizedSmithWatermanGotoh(std::string const& s, std::string const& t,
-                                    double gapValue = -0.5);
+float NormalizedSmithWatermanGotoh(std::string const& a, std::string const& b,
```

Comment:
`Similarity` is a `double`, keep the types. `MongeElkan` was changed to `float` for experiments, will be changed back.